### PR TITLE
docker: Improve complete README document

### PR DESCRIPTION
**Pull request by okurz**
* Fix typos, grammar, missing articles
* Consistently wrap at 78 characters
* Avoid contractions
* Consolidate on names, e.g. "openQA web UI"
* Add missing markup, e.g. for verbatim terms

### CODE REVIEW

**Comment by okurz** [comment]
Followup to https://github.com/os-autoinst/openQA/pull/3488

File: docker/README.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 the Fedora images or images built locally.
 
-The second one uses a docker-compose to deploy a complete web UI with HA for the
+The second one uses docker-compose to deploy a complete web UI with HA for the
 API and the dashboard.
 
-If a section or action is tagged as "Fedora", it shall be applied only for
-Fedora deployment. On the contrary, if the tag "docker-compose" is used it will
-be used for the docker-compose deployment.
+If a section or action is tagged as "Fedora", it shall be applied only for a
```
Comment (kalikiana) [inline docker/README.md:12]:
```suggestion
If a section or action is tagged as "Fedora", it will be applied only for a
```

**Review by kalikiana — APPROVED** [review]

**Comment by okurz** [comment]
@ilausuch would you like to review?

File: docker/README.md (hunk 1 above)
Comment (okurz) [reply docker/README.md:12]: That's not right. nothing "will be applied", the user has to do it.

Comment (kalikiana) [reply docker/README.md:12]: *shall* in the archaic sense means, it will happen. Perhaps what you meant was "should"?

**Review by ilausuch — APPROVED** [review]

File: docker/README.md [hunk 2]
```diff
@@ -207,65 +238,91 @@ And set permissions, so any user can read/write the data:
 
     chmod -R 777 data
 
-This step is unfortunately necessary because Docker [can't mount volume with specific user ownership](https://github.com/docker/docker/issues/7198) in container, so ownership of mounted folders (uid and gid) is the same as on your host system (presumably 1000:1000 which maps into nonexistent user in all of the containers).
+This step is unfortunately necessary because Docker
+[can not mount volume with specific user ownership](https://github.com/docker/docker/issues/7198)
```
Comment (Martchus) [inline docker/README.md:242]:
```suggestion
[can not mount a volume with specific user ownership](https://github.com/docker/docker/issues/7198)
```

File: docker/README.md [hunk 3]
```diff
@@ -207,65 +238,91 @@ And set permissions, so any user can read/write the data:
...
+your host system (presumably 1000:1000 which maps into nonexistent user in all
+of the containers).
 
-If you wish to keep the tests (for example) separate from the shared directory, for any reason (we do, in our development scenario) refer to the [Developing tests with Container setup] section at the end of this document.
+If you wish to keep the tests (for example) separate from the shared
+directory, for any reason (we do, in our development scenario) refer to the
+[Developing tests with Container setup] section at the end of this document.
 
-Populate the OpenQA's database:
+Populate the OpenQA database:
```
Comment (Martchus) [inline docker/README.md:251]:
```suggestion
Populate the openQA database:
```

**Review by Martchus — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 2 above)
Comment (okurz) [reply docker/README.md]: done

File: docker/README.md (hunk 3 above)
Comment (okurz) [reply docker/README.md]: done along with other references showing the same

File: docker/README.md (hunk 1 above)
Comment (okurz) [reply docker/README.md]: *I* did not mean anything, it was there in before. I have exchanged the word "shall" with "should" now to avoid the ambiguity.

**Review by Martchus — APPROVED** [review]

**Pull request merged**